model/registry/gen-ai.yaml

```diff
@@ -71,25 +71,21 @@ groups:
         brief: The number of tokens used in the LLM response (completion).
         examples: [180]
         tag: llm-generic-response
-      - id: usage.token_type
+      - id: token.type
         stability: experimental
         type:
-          allow_custom_values: true
+          allow_custom_values: false
```

Comment:
please keep `allow_custom_values: true` - the context https://github.com/open-telemetry/build-tools/issues/294 - we removed all other cases when it's `false` and will remove the flag alltogether